.github/workflows/pre-release.yml

```diff
@@ -22,38 +22,45 @@ jobs:
     name: Docker Images
     runs-on: ubuntu-latest
     needs: version
+    strategy:
+      matrix:
+        tags: ['', 'fips']
     env:
       DOCKER_USER: ${{ secrets.RELEASE_DOCKER_USER }}
       DOCKER_PASS: ${{ secrets.RELEASE_DOCKER_PASS }}
       VERSION: ${{ needs.version.outputs.version }}
       BUILD_DATE: "$$(date +%Y-%m-%d-%H:%M)"
+      FIPS: ${{ contains(matrix.tags, 'fips') && 1 || 0 }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Docker Login
         run: docker login --username "$DOCKER_USER" --password-stdin <<< "$DOCKER_PASS"
       - name: Push images with version tag
         env:
-          CTR_TAG: ${{ needs.version.outputs.version }}
+          CTR_TAG: ${{ contains(matrix.tags, 'fips') && format('{0}-fips', needs.version.outputs.version) || needs.version.outputs.version }}
```

Comment:
Here's a working nightly run: https://github.com/keithmattix/osm/actions/runs/2762068316

Working on the pre-release and release workflows

Reply:
@nojnhuh Turns out, my pre-release and image based runs are failing because I don't have a docker login and password in my fork: https://github.com/keithmattix/osm/actions/runs/2763317578

Reply:
Passing Pre-release workflow: https://github.com/openservicemesh/osm/blob/b1682d62f4828264328f8b7bc96a241e16aac749/pkg/identity/types.go#L16

Reply:
Passing release workflow: https://github.com/keithmattix/osm/actions/runs/2799846163